app/javascript/pages/IndividualEvents/index.js

```diff
@@ -378,6 +379,22 @@ const individualEventInputToOptimisticResponse = (data, input) => {
   }
 }
 
+const buildOptimisticResponse = (newEvent, currentUser) => {
+  const { id: currentUserId, individualEvents: existingEvents } = currentUser
+  const arrIdx = R.find(R.propEq('id', newEvent.id), existingEvents)
+  const individualEvents = R.isNil(arrIdx)
+    ? R.append(newEvent, existingEvents)
+    : R.update(arrIdx, newEvent, existingEvents)
+  return {
```

Comment:
would prefer a newline above the return, easier to read IMO

Reply:
Done.